### ISSUE TITLE
[React Native Gallery -> DatePicker]: Incorrect focus order is observed upon activating 'Next' or 'Previous' month buttons.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#511]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Keyboard
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘DatePicker’ button present under “Date & Time” group and invoke it.
Navigate to 'calendar date picker' button present under "A simple DatePicker" heading.
Navigate to 'Next' button and invoke it.
Now navigate using tab key and observe the issue.
Actual Result:
Incorrect focus order is observed upon activating 'Next' or 'Previous' month buttons.

Expected Result:
Proper focus order should be defined upon activating 'Next' or 'Previous' month buttons. When user hit tab key from 'Next' month button, keyboard focus should land on first date of selected month.

User Impact:
Users with motor disabilities who rely on keyboard will be impacted if incorrect focus order is defined.

Link to ADO bug: https://microsoft.visualstudio.com/OS/_workitems/edit/54110019
